### DISCUSSION TITLE
Edit 'LiquidMenu_config.h' file to change it.  errror

### DISCUSSION THREAD

**Discussion by kneo1** [VasilKalchev/LiquidMenu#88] [Q&A]
Hello. 
Newbie here. Trying some arduino projects. Building a menu with a rotary encoder and 20x4 lcd on a Nano 3, 328p old bootloader.
I have a working piece of code that prints some text on the lcd and  prints or removes "*****" as the encoder rotates, like a bar of progress sort of thing. 

I want to use LiquidMenu library and build a menu and just by #include <LiquidMenu.h> and nothing else , compiling shows many orange highlighted text similar to this:

`In file included from C:\Users\konst\Documents\Arduino\libraries\LiquidMenu-master\src\LiquidSystem.cpp:30:0:
C:\Users\konst\Documents\Arduino\libraries\LiquidMenu-master\src\LiquidMenu.h:52:123: note: #pragma message: LiquidMenu: Selected 'LiquidCrystal' (parallel) library. Edit 'LiquidMenu_config.h' file to change it.
 # pragma message ("LiquidMenu: Selected 'LiquidCrystal' (parallel) library. Edit 'LiquidMenu_config.h' file to change it.")
`
and this at the end:
`Could not find boards.txt in C:\Program Files (x86)\Arduino\hardware\sanguino\bootloaders. Is it pre-1.5?
Could not find boards.txt in C:\Program Files (x86)\Arduino\hardware\sanguino\cores. Is it pre-1.5?
Could not find boards.txt in C:\Program Files (x86)\Arduino\hardware\sanguino\firmwares. Is it pre-1.5?
Could not find boards.txt in C:\Program Files (x86)\Arduino\hardware\sanguino\variants. Is it pre-1.5?
`

Liquid menu is v1.6.0
The thing is that if I add #include <LiquidMenu.h> in my code and just upload it, my code does not work anymore, it prints the text on the lcd but the encoder wont read and push button does work but very very sluggyshly.
Is this a config error of some sort? or library installation
I  installed the LiquidMenu library from zip file. 

If I just comment out the #include <LiquidMenu.h>, the code will compile upload and work just fine.

Thank you.

**Comment by VasilKalchev** [comment]
Hello @kneo1 and sorry for the late reply.

The pragma message is there just as a notification.

> "Could not find boards.txt..."
> ...the encoder wont read and push button does work but very very sluggyshly.

This seems like a problem with the timer's configurations. I've experienced such problems when using a wrong bootloader or wrong Arduino compilation configuration. For simplicity, I would first try a fresh install of Arduino IDE (preferably not an old version) and upload the correct bootloader for the board you're using.
If this doesn't work, let me know if the unmodified examples from this library work correctly, the version of the Arduino IDE, the compilation configuration, the bootloader you are using and the frequency you are providing to the AVR.

Hope this helps and good luck.
